examples/fitness/lib/feed.dart

```diff
@@ -191,7 +191,7 @@ class FeedFragmentState extends State<FeedFragment> {
   }
 
   void _handleActionButtonPressed() {
-    showDialog(config.navigator, (NavigatorState navigator) => new AddItemDialog(navigator)).then((routeName) {
+    showDialog(context: context, child: new AddItemDialog()).then((routeName) {
```

Comment:
API suggestion: make the 'child' argument the first (positional) argument to showDialog(), and the context a named argument?